pkg/clients/rds/rds.go

```diff
@@ -185,5 +189,8 @@ func IsErrorNotFound(err error) bool {
 	if err == nil {
 		return false
 	}
+	if e, ok := err.(*sdkErrors.ServerError); ok && e.ErrorCode() == ErrCodeInstanceNotFound {
```

Comment:
Can you add a comment indicating that this happens in DeleteInstance()?

Reply:
Thanks, already add comment to this line.